cedar-drt/fuzz/src/schemas.rs

```diff
@@ -113,6 +113,12 @@ fn action_type_equivalence(name: &str, lhs: ActionType, rhs: ActionType) -> Resu
                     ))
                 }
             }
+            // An action w/ empty applies to list is equivalent to an action with _no_ applies to
+            // section at all.
+            // This is because neither action can be legally applied to any principal/resources.
+            (Some(applies_to), None) | (None, Some(applies_to)) if both_empty(&applies_to) => {
```

Comment:
I think you want an `either_empty` function here, because if either list is empty then the action applies to nothing.

Reply:
Oh yeah you're right